code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -230,7 +239,7 @@
 /mob/living/simple_animal/bot/cleanbot/ui_act(action, params)
 	if(..())
 		return
-	if(action != "area" && topic_denied(usr))
+	if(topic_denied(usr))
```

Comment:
Can you rvert this? the area action check is needed so that its public

Reply:
That's weird, I do not remember touching that part.. I guess I somehow downloaded an old file ? It is reverted.